terraform/modules/enclave/prometheus/prometheus.conf.tpl

```diff
@@ -15,7 +15,17 @@ scrape_configs:
     - source_labels: [__meta_ec2_tag_Name]
       regex: '.*\.stubs\..*'
       action: drop
-    - source_labels: [__meta_ec2_private_ip]
-      target_label: private_ip
+    - source_labels: [__meta_ec2_tag_Name]
+      target_label: instance
+
+  - job_name: 'hub_policy'
+    ec2_sd_configs:
```

Comment:
Missing `metrics_path` here (I snowflaked it and forgot to push to the branch)

Reply:
Good spot, will fix